models/index.js

```diff
@@ -0,0 +1,24 @@
+const sequelize = require('sequelize')
+const authorsModel = require('./authors')
+const novelsModel = require('./novels')
+const genresModel = require('./genres')
+const novelGenresModel = require('./novelGenres')
+
+const connection = new sequelize('novels', 'Alexandre', 'DUM@s!', {
+  host: 'localhost', dialect: 'mysql'
+})
```

Comment:
This should be reading from your config file. It actually caused every route to throw an error since it doesn't.